### PR TITLE
Issue/aztec media insertion

### PR DESCRIPTION
**Pull request by SergioEstevao**
This PR adds support to insert media on a post from the device or the media library.

Note: this PR doesn't support upload progress or check to see if media is still going on when closing post that will come on separate PRs.

To test:
 - Open the app
 - Activate the Aztec Editor on Me->Settings
 - Create a new Post
 - Add Media to it from the device or from the WordPress Media Library
 - Check if upload is done correctly

Needs review: @diegoreymendez

### CODE REVIEW

**Comment by astralbodies** [comment]
@SergioEstevao The unit tests failed to compile:

```
Testing failed:
    	Type 'AztecPostViewController.Constants' has no member 'defaultMissingImage'
    	Type 'AztecPostViewController.Constants' has no member 'defaultMissingImage'
    	Type 'AztecPostViewController.Constants' has no member 'defaultMissingImage'
    	Type 'AztecPostViewController.Constants' has no member 'defaultFont'
    ** TEST FAILED **
    The following build commands failed:
    	CompileSwift normal x86_64 /tmp/sandbox/workspace/WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift
    	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
```

**Comment by SergioEstevao** [comment]
@astralbodies just updated, something got wrong when I merged develop on this, but I fixed the code and pushed again.

File: WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift [hunk 1]
```diff
@@ -736,14 +740,13 @@ extension AztecPostViewController : Aztec.FormatBarDelegate {
...
-
     func showImagePicker() {
-        let picker = UIImagePickerController()
-        picker.sourceType = .photoLibrary
-        picker.mediaTypes = UIImagePickerController.availableMediaTypes(for: .photoLibrary) ?? []
+
+        let picker = WPMediaPickerViewController()
+        picker.dataSource = mediaLibraryDataSource
+        picker.showMostRecentFirst = true
+        picker.filter = WPMediaType.image
```
Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift]: (No need to change anything!). Swift's awesomeness: you can do this:

```
picker.filter = .image
```

**Comment by jleandroperez** [comment]
Code looks good! Testing on the actual device right away.

**Comment by jleandroperez** [comment]
- Uploads from the Device Library confirmed to be working great
- Uploads from the WordPress Media Lib establish a link to the original image (won't *reupload* the assets, that's expected, right?)

:shipit: once the conflicts are solved. Nice work sir!!

**Pull request merged**